src/pages/index.js

```diff
@@ -84,6 +110,37 @@ const handleImageClick = ({ title, image }) => {
   imagePopup.open({ title, image });
 };
 
+const handleCardDelete = (card) => {
+  confirmationPopups["delete-card-modal"].open();
+  return id;
+};
+
+const handleCardLike = (id, evt) => {
+  if (evt.target.classList.contains("locations__card-like_active")) {
```

Comment:
`  if (evt.target.classList.contains("locations__card-like_active")) {
`

Now only class `Card` knows every **card selector**, so you need to encapsulate the likes logics inside its methods and call them in `index.js`. It’s a violation of **Single responsibility** principle